src/components/CompareDiff.ts

```diff
@@ -7,7 +7,10 @@ export const CompareDiff = createDiffComponent({
     const container = this.getDeletionsElement()?.parentElement;
     container?.appendChild(spinner);
   },
-  getAdditionsText: (count) => `${count} additions`,
-  getDeletionsText: (count) => `${count} deletions`,
-  getGeneratedText: (count) => `${count} generated lines.`,
+  getAdditionsText: (count) =>
+    i18n.tp("diffs_additionsText", count, [String(count)]),
```

Comment:
I don't like that there's a separate function for plurals. I'd rather just use an override for `t` with 3 arguments.